sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,9 @@
+select count(*) as "Number of Movies",
+       "genres"."name" as "Genre"
+  from "genres"
+  join "filmGenre" using ("genreId")
+  join "films" using ("filmId")
+  join "castMembers" using ("filmId")
+  join "actors" using ("actorId")
+  where "actorId" = 178
```

Comment:
For this query, please restrict what information you use to that which you know ahead of time, without performing another query (manual or otherwise):

---

![image](https://user-images.githubusercontent.com/16143771/186547914-37236888-6f13-470b-8beb-c11afefc5ed0.png)